beetsplug/play.py

```diff
@@ -138,7 +138,7 @@ def play_music(self, lib, opts, args):
             open_args = [self._create_tmp_playlist(paths)]
 
         self._log.debug(u'executing command: {} {}', command_str,
-                        util.text_string(b' '.join(open_args)))
+                        ui.decargs(open_args))
```

Comment:
Since this is just a debug log, let's use `repr`. That is, we can use `{!r}` in the format string to get Unicode representations of arbitrary values.